packages/ui-components/src/icons.tsx

```diff
@@ -16,6 +16,7 @@
 
 import { LabIcon } from '@jupyterlab/ui-components';
 
+import addSnippetSvg from '../style/icons/addSnippet.svg';
```

Comment:
since this is a pretty generic icon that could be used elsewhere I would suggest not using "snippet" in the name

Reply:
that should have applied to our pipeline icons as well

Reply:
Do you think that [this line](https://github.com/elyra-ai/elyra/pull/589/files/ed11b87edce40a9277c9bc2d612a085c6192a56d#diff-075ae968e6c9905859aaa6eb9c77d6f4R34) makes it unique though? The variable name is just for retrieving it from the `style` directory.

Reply:
I meant all of it, the filename, vars names, style names. It's just an add icon, so it's name shouldn't include snippet. (Whereas the pipeline icons are a custom color for it's UI specifically)